### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by cuducos** [timofurrer/try#1]
The idea of _try_ is awesome, thanks! Unfortunately I was trying to try _try_  but I couldn't install it:

```console
$ python -V
Python 3.5.1
$ pip -V
pip 8.1.0 from /usr/local/var/pyenv/versions/3.5.1/lib/python3.5/site-packages (python 3.5)
$ pip install trypackage
Collecting trypackage
  Using cached trypackage-0.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/lr/gd0yq9m90rv_gz682gqt7ylc0000gn/T/pip-build-ftxztokx/trypackage/setup.py", line 44, in <module>
        install_requires=list(x.name for x in parse_requirements("./requirements.txt")),
      File "/private/var/folders/lr/gd0yq9m90rv_gz682gqt7ylc0000gn/T/pip-build-ftxztokx/trypackage/setup.py", line 44, in <genexpr>
        install_requires=list(x.name for x in parse_requirements("./requirements.txt")),
      File "/usr/local/var/pyenv/versions/3.5.1/lib/python3.5/site-packages/pip/req/req_file.py", line 79, in parse_requirements
        "parse_requirements() missing 1 required keyword argument: "
    TypeError: parse_requirements() missing 1 required keyword argument: 'session'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/lr/gd0yq9m90rv_gz682gqt7ylc0000gn/T/pip-build-ftxztokx/trypackage/
------------------------------------------------------------
$
```

Basically my setup is Mac OS X 11 (El Captain), using Python via pyenv.

**Comment by timofurrer** [comment]
Thanks for reporting :beers: 

The signature of `parse_requirements` seems to have changed in newer versions of pip.

I fixed it with 4dbbd7dcf3962dc02b3df47af5d0e29714e77675 and release **0.1.2**

**Comment by cuducos** [comment]
Awesome! Cheers : )

```console
$ pip install trypackage
Collecting trypackage
  Downloading trypackage-0.1.2.tar.gz
Collecting click (from trypackage)
  Using cached click-6.3-py2.py3-none-any.whl
Installing collected packages: click, trypackage
  Running setup.py install for trypackage ... done
Successfully installed click-6.3 trypackage-0.1.2
------------------------------------------------------------
$
```
